server/test/e2e/auth.e2e-spec.ts

```diff
@@ -64,27 +73,37 @@ describe(`${AuthController.name} (e2e)`, () => {
     it('should sign up the admin with a local domain', async () => {
       const { status, body } = await request(server)
         .post('/auth/admin-sign-up')
-        .send({ ...signupStub, email: 'admin@local' });
+        .send({ ...adminSignupStub, email: 'admin@local' });
       expect(status).toEqual(201);
       expect(body).toEqual({ ...signupResponseStub, email: 'admin@local' });
     });
 
     it('should transform email to lower case', async () => {
       const { status, body } = await request(server)
         .post('/auth/admin-sign-up')
-        .send({ ...signupStub, email: 'aDmIn@IMMICH.app' });
+        .send({ ...adminSignupStub, email: 'aDmIn@IMMICH.app' });
       expect(status).toEqual(201);
       expect(body).toEqual(signupResponseStub);
     });
 
     it('should not allow a second admin to sign up', async () => {
       await api.adminSignUp(server);
 
-      const { status, body } = await request(server).post('/auth/admin-sign-up').send(signupStub);
+      const { status, body } = await request(server).post('/auth/admin-sign-up').send(adminSignupStub);
 
       expect(status).toBe(400);
       expect(body).toEqual(errorStub.alreadyHasAdmin);
     });
+
+    it('should not allow null values', async () => {
+      for (const key of Object.keys(adminSignupStub)) {
```

Comment:
If you swap these two lines you can have it log the key name in the test description.

Reply:
Oh, good point